site/en/guides/border-router/raspberry-pi.md

```diff
@@ -0,0 +1,84 @@
+# Raspberry Pi
+
+OpenThread Border Router (OTBR) provides support for the [Raspberry Pi
+3B or newer](https://www.raspberrypi.org/products) (RPi)
+platform.
+
+Hardware requirements:
+
+*   External 5V AC adapter for power
+*   A supported [OpenThread platform](https://openthread.io/platforms) for Thread network
+    connectivity in an RCP design
+*   A microSD card and microSD card reader ("SD card" in this guide)
+
+Access the RPi with an external monitor (via HDMI) and keyboard (via USB), or
+connect a [serial cable](../../../guides/border-router/serial-cables.md) from the RPi to
+your computer and enable the serial console.
+
+To use RPi with OTBR:
+
+1.  [Download and install the OS](#download-and-install-the-os).
+1.  [Build and install OTBR](../../../guides/border-router/build/index.md).
```

Comment:
The local links for pages in the border-router/ directory should only need to go up 2, not 3